src/Microsoft.Tye.Core/ConfigModel/ConfigService.cs

```diff
@@ -24,5 +24,6 @@ public class ConfigService
         public List<ConfigVolume> Volumes { get; set; } = new List<ConfigVolume>();
         [YamlMember(Alias = "env")]
         public List<ConfigConfigurationSource> Configuration { get; set; } = new List<ConfigConfigurationSource>();
+        public List<ConfigConfigurationSource> Properties { get; set; } = new List<ConfigConfigurationSource>();
```

Comment:
Did you reuse configuration source because it exists?

Reply:
Yes

Reply:
I wouldn't. I would be specific about the intent of this field. It should be called buildProperties and should be a BuildProperty object

Reply:
Ok

Reply:
Fixed